app/templates/gentelella/admin/event/tickets/access_codes_create.html

```diff
@@ -227,9 +227,8 @@
 
         var host = (document.URL.split("//")[1].split("/"))[0];
         $("#code").change(function(){
-            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?code="+$(this).val());
+            $("#link").val(document.URL.split("/")[0]+"//"+host+"/e/"+"{{ event.identifier }}"+"/?access_code="+$(this).val());
```

Comment:
@magdalenesuo for now we cannot combine both access code and discount link in one code, this commit is only for event description page for now if the link is already provided to the user.
- And for making `acode=VIP&dcode=888` link while creating a discount code we need to provide a access code so that we can generate required link and for that we need to make  another commit